.travis.yml

```diff
@@ -1,5 +1,5 @@
 os: linux
-dist: trusty
+dist: xenial
```

Comment:
Isn't xenial 16.x?

Reply:
You are right, let's see if this fixes the issue, if it does I fix the commit message and the title.